src/foam/nanos/nanos.js

```diff
@@ -24,6 +24,16 @@ FOAM_FILES([
   { name: "foam/nanos/auth/Region" },
   { name: "foam/nanos/auth/Phone" },
   { name: "foam/nanos/auth/SignOutView" },
+  { name: "foam/nanos/auth/forgotPassword/EmailView" },
+  { name: "foam/nanos/auth/forgotPassword/ResendView" },
+  { name: "foam/nanos/auth/forgotPassword/ResetView" },
+  { name: "foam/nanos/auth/forgotPassword/SuccessView" },
+  { name: "foam/nanos/auth/forgotPassword/ResetPasswordController" },  
+  { name: "foam/nanos/auth/forgotPassword/ResetPasswordTokenService" },  
```

Comment:
This is just me being pedantic but I would change the folder to `foam/nanos/auth/resetPassword` just because you're using the word Reset in some of the class names.